### PR TITLE
{vis}[GCCcore/10.3.0,foss/2021a] NiBabel v3.2.1, pydicom v2.2.2 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by crubb**
(created using `eb --new-pr`)

... for @vacmar01 ;)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
node302.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/446adbf5f990f034f6c09e0d13c5d164 for a full test report.

File: easybuild/easyconfigs/n/NiBabel/NiBabel-3.2.1-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+homepage = 'https://nipy.github.io/nibabel'
+description = """NiBabel provides read/write access to some common medical and neuroimaging file formats,
+ including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT
+ as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and
+ annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
```
Comment (lexming) [inline easybuild/easyconfigs/n/NiBabel/NiBabel-3.2.1-foss-2021a.eb:15]: This dependency can be removed. NiBabel does not use/need CUDA.

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/n/NiBabel/NiBabel-3.2.1-foss-2021a.eb (hunk 1 above)
Comment (crubb) [reply easybuild/easyconfigs/n/NiBabel/NiBabel-3.2.1-foss-2021a.eb]: How did that sneak in there? ;) Thank you for the feedback. I have removed it!

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
centos7.lan - Linux UNKNOWN UNKNOWN, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/b74ca2ad99cb14c816c8cc65378a9805 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node316.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/308084520a54c27dbd54a95ed88a0d05 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @crubb !

**Pull request merged**